### DISCUSSION TITLE
Are contributions of layouts wanted?

### DISCUSSION THREAD

**Discussion by kylomax** [Obsidian-TTRPG-Community/fantasy-statblocks#79] [Q&A] [answered]
I'm aware of the layouts thread, I'm just wondering if contributing custom layouts I've made is wanted or not.

**Comment by valentine195** [comment, marked as answer]
Absolutely!

**Comment by kylomax** [comment]
Great, thanks :)

**Comment by BobHood** [comment]
Where would we Poste them?

**Reply by BobHood** [reply]
Found it...
Posted
